frontend/src/app/testQueue/QueueItem.scss

```diff
@@ -18,6 +18,8 @@
       font-size: 16px;
       font-weight: 400;
       color: #71767a;
+      min-width: 190px;
+      padding-top: 6px;
 
```

Comment:
Fix looks good for desktop and tablet but mobile view now is getting some items out of screen.
![Screenshot 2022-12-23 at 10 34 04 AM](https://user-images.githubusercontent.com/103958711/209361159-fc6b1874-0ad2-408b-9ded-0ef49d78770c.png)

For a comparison how it behaves in training with current code.

![Screenshot 2022-12-23 at 10 35 50 AM](https://user-images.githubusercontent.com/103958711/209361249-f8873247-f939-444b-a45b-0baf255ff094.png)

Reply:
I had some concerns about this too. Because we are not at all optimized for mobile across the ui, I am not sure how much effort we should put into optimizing for smaller than tablet. I'd defer to @kenieh on this.

Reply:
If we want to maintain exactly the same behavior as before, we can, it is just that dob will break onto multiple lines. @kenieh are you ok with that behavior? currently that is what happens anyway.

https://user-images.githubusercontent.com/89797785/209364313-69bd6595-6e19-4a86-bf53-0af63092b148.mov

Reply:
The other solution is we could put a media tag in for screen size and change the css when the screen moves to mobile size, but because we do not do that across the app, I would prefer not to do that here. I think we need more overall design before optimizing individual components for mobile behavior.

Reply:
For now, I have removed the `min-width` on the dob and left the existing behavior as suggested.

Reply:
I personally would rather have the DOB breaking into multiple lines than elements out of the screen. Thanks for looking into it!